.github/workflows/release.yml

```diff
@@ -226,11 +226,80 @@ jobs:
         with:
           path: contour-${{ steps.set_vars.outputs.version }}-${{ steps.set_vars.outputs.RUN_ID }}-osx.dmg
           retention-days: 30
+  build_archlinux:
+    strategy:
+      matrix:
+        arch:
+          [
+              "linux/amd64 x86_64",
+              "linux/arm64/v8 aarch64",
+          ]
+    name: "Archlinux ${{ matrix.arch }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+      - name: Read info
+        id: tags
+        shell: bash
+        run: |
+          arch="${{ matrix.arch }}"
+          echo ::set-output name=PLATFORM::${arch%% *}
+          echo ::set-output name=ARCH::${arch##* }
+      - name: set environment variables
+        id: set_vars
+        run: ./scripts/ci-set-vars.sh
+        env:
+          REPOSITORY: ${{ github.event.repository.name }}
+      - name: Build ${{ matrix.arch }} release
+        shell: bash
+        run: |
+          arch="${{ matrix.arch }}"
+          PLATFORM=${arch%% *}
+          ARCH=${arch##* }
+          mkdir artifacts
+          docker buildx build --platform $PLATFORM \
+          --tag contour:$ARCH \
+          -f archlinux/Dockerfile \
+          --load \
+          .
+          docker create --name contour-$ARCH contour:$ARCH
+          docker cp contour-$ARCH:/app/contour.tar.zst contour-$ARCH.tar.zst
+          docker container rm contour-$ARCH
+      - name: "test: libcrispy"
+        shell: bash
+        run: |
+          arch="${{ matrix.arch }}"
+          PLATFORM=${arch%% *}
+          ARCH=${arch##* }
+          docker run --rm contour:$ARCH build/src/crispy/crispy_test
+      - name: "test: libterminal"
+        shell: bash
+        run: |
+          arch="${{ matrix.arch }}"
+          PLATFORM=${arch%% *}
+          ARCH=${arch##* }
+          docker run --rm contour:$ARCH build/src/terminal/terminal_test
+      - name: "Uploading Archlinux .tar.zst package"
+        uses: actions/upload-artifact@v2
+        with:
+          name: contour_${{ steps.set_vars.outputs.version }}-${{ steps.set_vars.outputs.RUN_ID }}-archlinux_${{ steps.tags.outputs.ARCH }}
```

Comment:
With my limited knowledge of Arch Linux and my rusty VM I once prepared, how do I actually test-install such an `.tar.zstd` package the recommended way? (Googling for that didn't quite yield some satisfying results right now) :)

Reply:
This is just are tar that uses Zstandard for the compression so you can just unpack it and try to run the bin in there. 

This file is not for installing it that is what the PKGBUILD files would be fore in the our. The tar.zstd are just the Fileformat that is used for the packages that are in the normal Repositorys they changed it last year to use [zstd from xz](https://archlinux.org/news/now-using-zstandard-instead-of-xz-for-package-compression/).


When I would guess as when you look at the [content for example](https://mirrors.edge.kernel.org/archlinux/community/os/x86_64/aarch64-linux-gnu-gcc-10.2.0-1-x86_64.pkg.tar.zst) event the ones from arch them self they are just tar files and get extractet and pacman then does the rest.


So for having one that can just be installed you need to have the output of the patch they I send as they producte a file like I send in the example.